packages/kbn-es/src/utils/log.js

```diff
@@ -0,0 +1,6 @@
+const { createToolingLog } = require('@kbn/dev-utils');
+
+const log = createToolingLog('verbose');
```

Comment:
I realize how easy it is to do this way, but it would be great if we could pass an instance of the log into the Cluster class rather than it always just using this instance.

Reply:
I agree this would also solve the nested indentation for tests. Will make the change.

Reply:
c536ed5a92ff1d96009ac512aae6752f9f334c07